microsoft-365/managed-desktop/get-started/register-devices-partner.md

```diff
@@ -17,69 +17,40 @@ This topic describes the steps for Partners to follow to register devices. The p
-10. Select **Register devices**. The system will add the devices to your list of devices on the **Devices blade**, marked as **Registration Pending**. Registration typically takes less than 10 minutes, and when successful the device will show as **Ready for user** meaning it's ready and waiting for an end-user to start using.
-
-
-You can monitor the progress of device registration on the main **Microsoft Managed Desktop - Devices** page. Possible states reported there include:
-
-| State | Description |
-|---------------|-------------|
-| Registration pending | Registration is not done yet. Check back later. |
-| Registration failed | Registration could not be completed. Refer to [Troubleshooting device registration](register-devices-self.md#troubleshooting-device-registration) for more information. |
-| Ready for user | Registration succeeded and the device is now ready to be delivered to the end user. Microsoft Managed Desktop will guide them through first time set-up, so there’s no need for you to do any further preparations. |
-| Active | The device has been delivered to the end user and they have registered with your tenant. This also indicates that they are regularly using the device. |
-| Inactive | The device has been delivered to the end user and they have registered with your tenant. However, they have not used the device recently (in the last 7 days).  |
-
-
-
-## Troubleshooting
-
-| Error message | Details |
-|---------------|-------------|
-| Device not found | We couldn’t register this device because we could not find a match for the provided manufacturer, model, or serial number. Confirm these values with your device supplier. |
-| Hardware hash not valid | The hardware hash you provided for this device was not formatted correctly. Double-check the hardware hash and then resubmit. |
-| Device already registered | This device is already registered to your organization. No further action required. |
-| Device claimed by another organization | This device has already been claimed by another organization. Check with your device supplier. |
-| Unexpected error | Your request could not be automatically processed. Contact Support (<support link>) and provide the Request ID: <requestId> |
+Before completing registration for a customer, you must first establish a relationship with them. You should have a unique link to provide to your respective customers. See [How to establish OEM relationship] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#oem-authorization).
+
+Once you've established the relationship, you can start registering devices for customers using the Group Tag **Microsoft365Managed_Autopilot**.
+
+> [!IMPORTANT]
+> The group name must **Microsoft365Managed_Autopilot** exactly, including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```

Comment:
```suggestion
> The group name must match **Microsoft365Managed_Autopilot** exactly, including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```